telethon_generator/data/errors.csv

```diff
@@ -271,6 +271,7 @@ START_PARAM_INVALID,400,Start parameter invalid
 STATS_MIGRATE_X,303,The channel statistics must be fetched from DC {dc}
 STICKERSET_INVALID,400,The provided sticker set is invalid
 STICKERS_EMPTY,400,No sticker provided
+STICKER_DOCUMENT_INVALID,400,Sticker document invalid
```

Comment:
This should have more information.

```suggestion
STICKER_DOCUMENT_INVALID,400,"The sticker file was invalid (make sure to use the correct format, animated stickers may not be allowed in this request)"
```

Reply:
Through some additional research I discovered that this error happens when the sticker doesn't comply with TG internal checks, mainly because it has too mane shapes or the geometry is too complex. Maybe have to rephrase the error description, because animated stickers are definitely allowed in requests.